common/Constants/Constants.py

```diff
@@ -0,0 +1 @@
+SATOSHI_IN_1_BTC = 100_000_000

```

Comment:
Could you please add sizes in bytes for input, output and container. 
I'd like to see them as constants for now, even they can be changed further.